pkg/kv/kvserver/scheduler_test.go

```diff
@@ -249,9 +249,14 @@ func TestSchedulerLoop(t *testing.T) {
 	m := newStoreMetrics(metric.TestSampleInterval)
 	p := newTestProcessor()
 	s := newRaftScheduler(log.MakeTestingAmbientContext(stopper.Tracer()), m, p, 1, 1)
-
 	s.Start(stopper)
-	s.EnqueueRaftTicks(1, 2, 3)
+
+	batch := s.NewEnqueueBatch()
+	defer batch.Close()
+	batch.Add(1)
+	batch.Add(2)
+	batch.Add(3)
```

Comment:
How about a variadic `Add`? OTOH, fine as is, since the prod code `Add`s one by one, so it's better to avoid hidden slices in it.

Reply:
Doesn't seem worth it.